src/server/test-suites/transformers/test-suite-status.js

```diff
@@ -42,10 +42,13 @@ function testSuiteStatus(service) {
     createRepository: {
       name: 'GitHub Repository',
       part: 1,
-      url: {
-        text: removeUrlParts(service?.githubUrl),
-        href: service?.githubUrl
-      },
+      url:
+        createRepository?.status === creationStatuses.success
+          ? {
+              text: `${serviceStatus.org}/${serviceStatus.repositoryName}`,
+              href: `https://github.com/${serviceStatus.org}/${serviceStatus.repositoryName}`
+            }
+          : { text: '', href: '' },
```

Comment:
can this just be null? and guard whatever is consuming it? `url?.text`? `url?.href`?